zebra-chain/src/block/tests/vectors.rs

```diff
@@ -213,7 +213,7 @@ fn block_test_vectors_height(network: Network) {
             zebra_test::vectors::MAINNET_BLOCKS.iter(),
             zebra_test::vectors::MAINNET_FINAL_SAPLING_ROOTS.clone(),
         ),
-        Testnet => (
+        Testnet(_) => (
```

Comment:
I don't think we should be matching on `Mainnet` or `Testnet` directly, because it's only going to make these kinds of changes harder. Every time we change the structure of the `Network` enum, we'll have to change all 400 lines across 50+ files.

Instead, let's create an `is_mainnet()` method, switch all this code to it in a PR, then implement the rest of this PR on top of that PR?

Reply:
> Instead, let's create an is_mainnet() method, switch all this code to it in a PR, then implement the rest of this PR on top of that PR?

Tsk. I added that method and forgot to use it, I'll fix that.

Reply:
This method already exists:
https://github.com/ZcashFoundation/zebra/blob/2efc6bf4b2875f16a987bcbff0f90cff98eda3fa/zebra-chain/src/parameters/network.rs#L129